.github/workflows/default.yml

```diff
@@ -54,12 +23,10 @@ jobs:
         run: |
           source ./venv/bin/activate
           inv test.hooks
-      - name: Poetry
-        uses: abatilo/actions-poetry@v2
       - name: Run tests on building project
         run: |
           source ./venv/bin/activate
           cookiecutter --no-input --overwrite-if-exists --output-dir build .
           cd build/awesome
-          make api-install
+          cp ./api/.env.example ./api/.env
           make api-test
```

Comment:
here we run the make command inside the GHA job that runs the docker image with docker-compose and run the make command inside the image.
it doesn't look really stable :)